sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -103,7 +103,7 @@
         }
       ],
       "content": "<span class=\"mb-2\">Study period</span> <span class=\"fa fa-circle formio-status-chip ml-1 mt-1 {{ data.offeringChipStatus }}\">{{ data.offeringStatusToDisplay }}</span>",
-      "refreshOnChange": true,
+      "refreshOnChange": false,

```

Comment:
shouldn't this be true

Reply:
`"refreshOnChange": true` was removing the value. This value does not change after it is loaded. I tested with and without a local cache. I can double-check.